database/sqlite3/sqlite3_test.go

```diff
@@ -181,3 +182,35 @@ func TestMigrateWithDirectoryNameContainsWhitespaces(t *testing.T) {
 	}
 	dt.Test(t, d, []byte("CREATE TABLE t (Qty int, Name string);"))
 }
+
+var pathTests = []struct {
+	in  string
```

Comment:
Add a test name (e.g. the test description you used in comments) and use that in `t.Run()`

Reply:
Done